Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -489,18 +515,35 @@ class DropDownMenu
     }
   }
 
-  private static ApplyPosition (popUpDiv: HTMLDivElement, clickEvent: Nullable<MouseEvent>, referenceElement: HTMLElement): void
+  private static ApplyPosition (popUpDiv: HTMLDivElement, clickEvent: Nullable<MouseEvent>, referenceElement: HTMLElement, isPeriodic: boolean): void
   {
     var space_top = Math.round (LayoutUtility.GetOffset (referenceElement).top - window.pageYOffset);
     var space_bottom = Math.round (document.documentElement.clientHeight - LayoutUtility.GetOffset (referenceElement).top - LayoutUtility.GetHeight (referenceElement) + window.pageYOffset);
     var space_left = LayoutUtility.GetOffset (referenceElement).left;
     var space_right = document.documentElement.clientWidth - LayoutUtility.GetOffset (referenceElement).left - LayoutUtility.GetWidth (referenceElement);
 
+    // Periodic update: only update our position if it has change to allow the usage of the scrollbars
+    // Other update: it might take a periodic call to get the correct position so we force it by deleting the last position information
+    if (isPeriodic) {
+      var positionString = `${space_top};${space_right};${space_bottom};${space_left};`;
+      if (positionString === this._lastPositionInformation)
+        return;
+
+      this._lastPositionInformation = positionString;
+    } else {
+      this._lastPositionInformation = null
+    }
+
     // position drop-down list
     var top = clickEvent ? clickEvent.clientY : Math.max (0, space_top + referenceElement.offsetHeight);
     var left = clickEvent ? clickEvent.clientX + 'px' : 'auto';
     var right = clickEvent ? 'auto' : Math.max (0, document.documentElement.clientWidth - LayoutUtility.GetOffset (referenceElement).left - referenceElement.offsetWidth) + 'px';
 
+    const verticalSpacing = parseInt(getComputedStyle(popUpDiv).fontSize) * 2;
```

Comment:
I would prefer to use the height of the first menu item (no factor), not simply the font size. if we also want to support line-wrapped items, we could take the minimum value for all menu items. That way, we have a distance that's more closely aligned with the layout.

Reply:
I don't think that that is a good idea as the size of the first element might change depending on the layout of popup which might cause flickering between states. I don't think that it is necessary here to have a spacing that is most closely aligned to the rest of the layout.

Reply:
hmm...yes, preventing this flicker on edge cases is important and would make things more complicated. Okay, we keep fontsize, but please add a comment about why we don't use itemheight to the source code.